Wikipedia/Code/OnThisDayCollectionViewCell.swift

```diff
@@ -1,16 +1,41 @@
 import UIKit
 
-@objc(WMFOnThisDayCollectionViewCell)
-class OnThisDayCollectionViewCell: SideScrollingCollectionViewCell {
-
-    let timelineView = OnThisDayTimelineView()
+@objc(WMFOnThisDayExploreCollectionViewCell)
+class OnThisDayExploreCollectionViewCell: OnThisDayCollectionViewCell {
 
-    @objc(configureForExploreWithOnThisDayEvent:previousEvent:dataStore:layoutOnly:)
-    func configureForExplore(with onThisDayEvent: WMFFeedOnThisDayEvent,  previousEvent: WMFFeedOnThisDayEvent, dataStore: MWKDataStore, layoutOnly: Bool) {
+    @objc(configureWithOnThisDayEvent:previousEvent:dataStore:layoutOnly:)
+    func configure(with onThisDayEvent: WMFFeedOnThisDayEvent,  previousEvent: WMFFeedOnThisDayEvent, dataStore: MWKDataStore, layoutOnly: Bool) {
```

Comment:
@montehurd just for the configuration methods - basically don't have those methods that take model objects in the main view files.